java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2526,6 +2526,18 @@ public final ColumnVector getMapValue(Scalar key) {
     return new ColumnVector(mapLookup(getNativeView(), key.getScalarHandle()));
   }
 
+  /** For a column of type List<Struct<String, String>> and a passed in String key, return a boolean
+   * scalar for all keys in the structs, true if the key exists in all maps in the column.
+   * @param key the String scalar to lookup in the column
+   * @return a boolean scalar based on the lookup result
+   */
+  public final Scalar getMapKeyExistence(Scalar key) {
```

Comment:
Why does this return a scalar instead of a column showing the row-by-row existence result?  By forcing the reduction, users can't use this in a case where they just want the boolean result by row.  If a particular use-case desires the reduction, let them call `all()` on the result rather than assume all use cases will want the reduction.